pkg/nsx/services/realizestate/realize_state.go

```diff
@@ -61,6 +61,11 @@ func (service *RealizeStateService) CheckRealizeState(backoff wait.Backoff, inte
 					if alarm.Message != nil {
 						errMsg = append(errMsg, *alarm.Message)
 					}
+					if alarm.ErrorDetails != nil {
+						for _, relatedErr := range alarm.ErrorDetails.RelatedErrors {
+							errMsg = append(errMsg, *relatedErr.ErrorMessage)
```

Comment:
better check if ErrorMessage is nil pointer

Reply:
Updated, thanks!